pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation_test.go

```diff
@@ -801,6 +806,65 @@ func TestInjectLibInitContainer(t *testing.T) {
 			wantErr: true,
 			wantCPU: "50m",
 			wantMem: "20Mi",
+			secCtx:  &corev1.SecurityContext{},
+		},
+		{
+			name:    "with full security context",
+			pod:     common.FakePod("java-pod"),
+			image:   "gcr.io/datadoghq/dd-lib-java-init:v1",
+			lang:    java,
+			wantErr: false,
+			wantCPU: "50m",
+			wantMem: "20Mi",
+			secCtx: &corev1.SecurityContext{
+				Capabilities: &corev1.Capabilities{
+					Add:  []corev1.Capability{"NET_ADMIN", "SYS_TIME"},
+					Drop: []corev1.Capability{"ALL"},
+				},
+				Privileged: pointerOf(false),
```

Comment:
```suggestion
				Privileged: pointer.Ptr(false),
```

For all of these, we already have `pointer` imported into this package, so we can drop the `pointerOf` function and use that instead

Reply:
Ah! Perfect, I tried to find something like this, but I'm not too familiar with Go yet 😅
Thank you - I've fixed it 😄 

And thanks for the review 🙏